bhr.js

```diff
@@ -468,16 +507,54 @@ function setSelectedRow(row) {
         '<span class="highlight">' + escape(string.slice(index, index + filterString.length)) + "</span>" +
         escape(string.slice(index + filterString.length));
     }
-    let annotation = "";
+    let bugzillaAnnotationHtml = "";
     if (row.hang.knownBug) {
-      annotation =
-        `<div id="bugzilla-annotation">
+      bugzillaAnnotationHtml =
+        `<div id="bugzilla-annotation" class="extra-hang-info">
            <span id="annotation-label">Bugzilla annotation:</span><ul>${
              row.hang.knownBug.signatures.map(a => "<li>" + escape(a) + "</li>").join("")
            }</ul>
          </div>`;
     }
-    div.innerHTML = `${annotation}<ul>${
+    let hangAnnotationHtml = "";
+    let hangAnnotations = [...Object.entries(row.hang.annotationStats)];
+    hangAnnotations.sort((a, b) => b[1]._totalCount - a[1]._totalCount);
+    if (hangAnnotations.length) {
+      let annotationStrings = hangAnnotations.map(([key, value]) => {
+        let values = [...Object.entries(value)].filter(([k, v]) => !k.startsWith("_"));
+        if (values.length == 1 && values[0][0] == "true") {
+          let annotationCount = values[0][1];
+          let annotationPercent = (annotationCount / row.hang.count).toLocaleString(
+            undefined,
+            { style: 'percent', minimumFractionDigits: 1 }
+          );
+          return `<li><code>${escape(key)}</code> : ${annotationPercent} (${annotationCount} hangs)</li>`;
+        } else if (values.length == 1) {
+          let valueKey = values[0][0];
+          let annotationCount = values[0][1];
+          let annotationPercent = (annotationCount / row.hang.count).toLocaleString(
+            undefined,
+            { style: 'percent', minimumFractionDigits: 1 }
+          );
+          return `<li><code>${escape(key)}: ${escape(valueKey)}</code> : ${annotationPercent} (${annotationCount} hangs)</li>`;
```

Comment:
The result of this is:
 `browser.tabs.opening: animated` : 0.0% (25 hangs)
I would prefer:
  `browser.tabs.opening`: 0.0% (25 hangs: `animated`)
I think this would be more consistent with the display in the other case.